package test.data;

import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class RegressionTest1 {

    public static boolean debug = false;

    public void assertBooleanArrayEquals(boolean[] expectedArray, boolean[] actualArray) {
        if (expectedArray.length != actualArray.length) {
            throw new AssertionError("Array lengths differ: " + expectedArray.length + " != " + actualArray.length);
        }
        for (int i = 0; i < expectedArray.length; i++) {
            if (expectedArray[i] != actualArray[i]) {
                throw new AssertionError("Arrays differ at index " + i + ": " + expectedArray[i] + " != " + actualArray[i]);
            }
        }
    }

    @Test
    public void test0501() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0501");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0502() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0502");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0503() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0503");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0504() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0504");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0505() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0505");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0506() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0506");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0507() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0507");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0508() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0508");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0509() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0509");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0510() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0510");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0511() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0511");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0512() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0512");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0513() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0513");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0514() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0514");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0515() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0515");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0516() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0516");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0517() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0517");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0518() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0518");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0519() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0519");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0520() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0520");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0521() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0521");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0522() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0522");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0523() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0523");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0524() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0524");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0525() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0525");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0526() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0526");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0527() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0527");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0528() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0528");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0529() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0529");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0530() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0530");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0531() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0531");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0532() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0532");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0533() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0533");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0534() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0534");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0535() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0535");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0536() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0536");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0537() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0537");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0538() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0538");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0539() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0539");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0540() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0540");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0541() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0541");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0542() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0542");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0543() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0543");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0544() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0544");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0545() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0545");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0546() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0546");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0547() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0547");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0548() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0548");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0549() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0549");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0550() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0550");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0551() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0551");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0552() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0552");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0553() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0553");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0554() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0554");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0555() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0555");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0556() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0556");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0557() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0557");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0558() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0558");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0559() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0559");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0560() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0560");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0561() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0561");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0562() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0562");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0563() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0563");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0564() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0564");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0565() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0565");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0566() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0566");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0567() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0567");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0568() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0568");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0569() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0569");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0570() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0570");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0571() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0571");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0572() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0572");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0573() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0573");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0574() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0574");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0575() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0575");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0576() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0576");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0577() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0577");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0578() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0578");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0579() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0579");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0580() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0580");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0581() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0581");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0582() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0582");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0583() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0583");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0584() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0584");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0585() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0585");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0586() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0586");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0587() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0587");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0588() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0588");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0589() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0589");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0590() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0590");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0591() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0591");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0592() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0592");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0593() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0593");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0594() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0594");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0595() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0595");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0596() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0596");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0597() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0597");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0598() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0598");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0599() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0599");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0600() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0600");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0601() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0601");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0602() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0602");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0603() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0603");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0604() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0604");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0605() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0605");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0606() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0606");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0607() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0607");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0608() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0608");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0609() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0609");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
    }

    @Test
    public void test0610() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0610");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0611() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0611");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0612() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0612");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0613() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0613");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean45 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass46 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertNotNull(wildcardClass46);
    }

    @Test
    public void test0614() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0614");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0615() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0615");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0616() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0616");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0617() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0617");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0618() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0618");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
    }

    @Test
    public void test0619() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0619");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0620() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0620");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0621() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0621");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0622() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0622");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0623() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0623");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean42 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass43 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertNotNull(wildcardClass43);
    }

    @Test
    public void test0624() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0624");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0625() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0625");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0626() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0626");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0627() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0627");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("", "");
        boolean boolean42 = userRepository0.validateUser("hi!", "");
        boolean boolean45 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
    }

    @Test
    public void test0628() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0628");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0629() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0629");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0630() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0630");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0631() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0631");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0632() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0632");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0633() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0633");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0634() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0634");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0635() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0635");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0636() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0636");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0637() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0637");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0638() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0638");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0639() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0639");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0640() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0640");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0641() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0641");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0642() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0642");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0643() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0643");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0644() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0644");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0645() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0645");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0646() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0646");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0647() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0647");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
    }

    @Test
    public void test0648() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0648");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0649() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0649");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0650() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0650");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0651() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0651");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0652() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0652");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0653() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0653");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0654() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0654");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0655() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0655");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0656() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0656");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0657() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0657");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0658() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0658");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0659() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0659");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0660() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0660");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0661() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0661");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0662() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0662");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0663() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0663");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0664() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0664");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass43 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertNotNull(wildcardClass43);
    }

    @Test
    public void test0665() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0665");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0666() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0666");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0667() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0667");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0668() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0668");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0669() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0669");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0670() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0670");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0671() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0671");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0672() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0672");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0673() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0673");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0674() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0674");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0675() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0675");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0676() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0676");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0677() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0677");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0678() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0678");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0679() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0679");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0680() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0680");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0681() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0681");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0682() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0682");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0683() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0683");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0684() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0684");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0685() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0685");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0686() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0686");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0687() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0687");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0688() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0688");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0689() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0689");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
    }

    @Test
    public void test0690() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0690");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0691() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0691");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0692() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0692");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0693() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0693");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0694() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0694");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0695() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0695");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0696() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0696");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0697() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0697");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0698() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0698");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0699() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0699");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0700() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0700");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0701() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0701");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0702() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0702");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0703() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0703");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0704() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0704");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0705() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0705");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0706() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0706");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0707() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0707");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0708() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0708");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0709() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0709");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0710() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0710");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0711() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0711");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0712() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0712");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0713() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0713");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("", "");
        boolean boolean42 = userRepository0.validateUser("hi!", "");
        boolean boolean45 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass46 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertNotNull(wildcardClass46);
    }

    @Test
    public void test0714() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0714");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0715() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0715");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0716() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0716");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0717() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0717");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0718() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0718");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0719() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0719");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0720() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0720");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0721() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0721");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0722() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0722");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0723() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0723");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0724() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0724");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0725() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0725");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0726() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0726");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0727() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0727");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0728() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0728");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass10 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertNotNull(wildcardClass10);
    }

    @Test
    public void test0729() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0729");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0730() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0730");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0731() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0731");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0732() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0732");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0733() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0733");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0734() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0734");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0735() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0735");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0736() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0736");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0737() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0737");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0738() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0738");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0739() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0739");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0740() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0740");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0741() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0741");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0742() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0742");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "");
        boolean boolean42 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
    }

    @Test
    public void test0743() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0743");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0744() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0744");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0745() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0745");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0746() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0746");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0747() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0747");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0748() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0748");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0749() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0749");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0750() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0750");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0751() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0751");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0752() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0752");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0753() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0753");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0754() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0754");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0755() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0755");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0756() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0756");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0757() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0757");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0758() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0758");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0759() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0759");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0760() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0760");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
    }

    @Test
    public void test0761() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0761");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0762() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0762");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0763() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0763");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0764() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0764");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0765() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0765");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0766() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0766");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0767() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0767");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0768() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0768");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0769() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0769");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
    }

    @Test
    public void test0770() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0770");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
    }

    @Test
    public void test0771() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0771");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0772() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0772");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0773() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0773");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0774() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0774");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0775() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0775");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0776() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0776");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0777() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0777");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0778() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0778");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0779() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0779");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0780() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0780");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0781() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0781");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0782() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0782");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0783() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0783");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0784() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0784");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0785() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0785");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0786() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0786");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0787() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0787");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0788() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0788");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0789() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0789");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0790() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0790");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0791() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0791");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0792() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0792");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("", "");
        boolean boolean42 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
    }

    @Test
    public void test0793() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0793");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0794() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0794");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0795() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0795");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0796() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0796");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0797() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0797");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0798() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0798");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0799() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0799");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0800() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0800");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0801() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0801");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0802() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0802");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0803() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0803");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0804() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0804");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0805() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0805");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0806() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0806");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass4 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertNotNull(wildcardClass4);
    }

    @Test
    public void test0807() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0807");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0808() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0808");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0809() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0809");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0810() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0810");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0811() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0811");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0812() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0812");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0813() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0813");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0814() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0814");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass7 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertNotNull(wildcardClass7);
    }

    @Test
    public void test0815() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0815");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0816() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0816");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0817() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0817");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0818() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0818");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0819() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0819");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0820() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0820");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0821() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0821");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0822() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0822");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0823() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0823");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0824() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0824");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0825() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0825");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0826() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0826");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0827() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0827");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0828() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0828");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0829() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0829");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0830() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0830");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0831() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0831");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0832() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0832");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0833() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0833");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0834() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0834");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0835() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0835");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0836() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0836");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0837() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0837");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0838() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0838");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0839() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0839");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0840() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0840");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0841() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0841");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0842() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0842");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0843() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0843");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0844() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0844");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0845() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0845");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0846() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0846");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0847() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0847");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0848() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0848");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0849() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0849");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0850() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0850");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0851() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0851");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0852() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0852");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0853() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0853");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0854() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0854");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0855() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0855");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0856() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0856");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean42 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass43 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertNotNull(wildcardClass43);
    }

    @Test
    public void test0857() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0857");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "");
        boolean boolean42 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
    }

    @Test
    public void test0858() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0858");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0859() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0859");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0860() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0860");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0861() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0861");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0862() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0862");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0863() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0863");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0864() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0864");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0865() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0865");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0866() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0866");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "");
        boolean boolean42 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
    }

    @Test
    public void test0867() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0867");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0868() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0868");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0869() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0869");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass7 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertNotNull(wildcardClass7);
    }

    @Test
    public void test0870() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0870");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0871() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0871");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass10 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertNotNull(wildcardClass10);
    }

    @Test
    public void test0872() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0872");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0873() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0873");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0874() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0874");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0875() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0875");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
    }

    @Test
    public void test0876() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0876");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0877() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0877");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0878() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0878");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0879() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0879");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0880() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0880");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0881() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0881");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0882() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0882");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0883() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0883");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0884() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0884");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0885() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0885");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0886() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0886");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0887() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0887");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0888() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0888");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0889() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0889");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0890() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0890");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0891() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0891");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0892() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0892");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
    }

    @Test
    public void test0893() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0893");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0894() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0894");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0895() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0895");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0896() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0896");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0897() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0897");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0898() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0898");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0899() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0899");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0900() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0900");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0901() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0901");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0902() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0902");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0903() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0903");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0904() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0904");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
    }

    @Test
    public void test0905() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0905");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0906() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0906");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0907() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0907");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0908() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0908");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean45 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass46 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertNotNull(wildcardClass46);
    }

    @Test
    public void test0909() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0909");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0910() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0910");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0911() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0911");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0912() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0912");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0913() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0913");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0914() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0914");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0915() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0915");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0916() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0916");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0917() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0917");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean45 = userRepository0.validateUser("", "hi!");
        boolean boolean48 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass49 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertTrue("'" + boolean48 + "' != '" + false + "'", boolean48 == false);
        org.junit.Assert.assertNotNull(wildcardClass49);
    }

    @Test
    public void test0918() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0918");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0919() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0919");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0920() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0920");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0921() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0921");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0922() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0922");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0923() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0923");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0924() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0924");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0925() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0925");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0926() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0926");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0927() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0927");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass43 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertNotNull(wildcardClass43);
    }

    @Test
    public void test0928() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0928");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0929() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0929");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0930() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0930");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0931() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0931");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0932() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0932");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0933() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0933");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0934() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0934");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0935() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0935");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0936() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0936");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass16 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertNotNull(wildcardClass16);
    }

    @Test
    public void test0937() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0937");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0938() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0938");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0939() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0939");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0940() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0940");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass43 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertNotNull(wildcardClass43);
    }

    @Test
    public void test0941() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0941");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass43 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertNotNull(wildcardClass43);
    }

    @Test
    public void test0942() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0942");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0943() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0943");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0944() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0944");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0945() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0945");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0946() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0946");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0947() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0947");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0948() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0948");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0949() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0949");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0950() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0950");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0951() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0951");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0952() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0952");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "");
        boolean boolean39 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
    }

    @Test
    public void test0953() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0953");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0954() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0954");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0955() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0955");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0956() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0956");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
    }

    @Test
    public void test0957() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0957");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0958() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0958");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean45 = userRepository0.validateUser("", "hi!");
        boolean boolean48 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass49 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertTrue("'" + boolean48 + "' != '" + false + "'", boolean48 == false);
        org.junit.Assert.assertNotNull(wildcardClass49);
    }

    @Test
    public void test0959() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0959");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0960() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0960");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0961() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0961");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0962() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0962");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass37 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertNotNull(wildcardClass37);
    }

    @Test
    public void test0963() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0963");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0964() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0964");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean45 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass46 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertNotNull(wildcardClass46);
    }

    @Test
    public void test0965() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0965");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0966() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0966");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0967() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0967");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0968() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0968");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }

    @Test
    public void test0969() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0969");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0970() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0970");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0971() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0971");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
    }

    @Test
    public void test0972() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0972");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0973() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0973");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0974() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0974");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0975() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0975");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass22 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertNotNull(wildcardClass22);
    }

    @Test
    public void test0976() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0976");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass13 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertNotNull(wildcardClass13);
    }

    @Test
    public void test0977() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0977");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0978() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0978");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0979() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0979");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0980() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0980");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        boolean boolean36 = userRepository0.validateUser("", "hi!");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass40 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertNotNull(wildcardClass40);
    }

    @Test
    public void test0981() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0981");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0982() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0982");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass28 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertNotNull(wildcardClass28);
    }

    @Test
    public void test0983() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0983");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass19 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertNotNull(wildcardClass19);
    }

    @Test
    public void test0984() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0984");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
    }

    @Test
    public void test0985() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0985");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0986() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0986");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0987() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0987");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        boolean boolean33 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass34 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertNotNull(wildcardClass34);
    }

    @Test
    public void test0988() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0988");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0989() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0989");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean45 = userRepository0.validateUser("", "hi!");
        boolean boolean48 = userRepository0.validateUser("", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertTrue("'" + boolean48 + "' != '" + false + "'", boolean48 == false);
    }

    @Test
    public void test0990() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0990");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean12 = userRepository0.validateUser("hi!", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test0991() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0991");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0992() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0992");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("hi!", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
    }

    @Test
    public void test0993() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0993");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0994() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0994");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean9 = userRepository0.validateUser("", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
    }

    @Test
    public void test0995() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0995");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "");
        boolean boolean27 = userRepository0.validateUser("hi!", "");
        boolean boolean30 = userRepository0.validateUser("", "hi!");
        boolean boolean33 = userRepository0.validateUser("", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "");
        boolean boolean39 = userRepository0.validateUser("", "hi!");
        boolean boolean42 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean45 = userRepository0.validateUser("", "hi!");
        boolean boolean48 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean51 = userRepository0.validateUser("", "");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
        org.junit.Assert.assertTrue("'" + boolean39 + "' != '" + false + "'", boolean39 == false);
        org.junit.Assert.assertTrue("'" + boolean42 + "' != '" + false + "'", boolean42 == false);
        org.junit.Assert.assertTrue("'" + boolean45 + "' != '" + false + "'", boolean45 == false);
        org.junit.Assert.assertTrue("'" + boolean48 + "' != '" + false + "'", boolean48 == false);
        org.junit.Assert.assertTrue("'" + boolean51 + "' != '" + false + "'", boolean51 == false);
    }

    @Test
    public void test0996() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0996");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        java.lang.Class<?> wildcardClass25 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertNotNull(wildcardClass25);
    }

    @Test
    public void test0997() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0997");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "");
        boolean boolean27 = userRepository0.validateUser("", "hi!");
        boolean boolean30 = userRepository0.validateUser("", "");
        boolean boolean33 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean36 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertTrue("'" + boolean33 + "' != '" + false + "'", boolean33 == false);
        org.junit.Assert.assertTrue("'" + boolean36 + "' != '" + false + "'", boolean36 == false);
    }

    @Test
    public void test0998() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0998");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("", "");
        boolean boolean9 = userRepository0.validateUser("", "hi!");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean18 = userRepository0.validateUser("", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
    }

    @Test
    public void test0999() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test0999");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean21 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean24 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        boolean boolean30 = userRepository0.validateUser("hi!", "");
        java.lang.Class<?> wildcardClass31 = userRepository0.getClass();
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
        org.junit.Assert.assertTrue("'" + boolean30 + "' != '" + false + "'", boolean30 == false);
        org.junit.Assert.assertNotNull(wildcardClass31);
    }

    @Test
    public void test1000() throws Throwable {
        if (debug)
            System.out.format("%n%s%n", "RegressionTest1.test1000");
        data.UserRepository userRepository0 = new data.UserRepository();
        boolean boolean3 = userRepository0.validateUser("hi!", "");
        boolean boolean6 = userRepository0.validateUser("hi!", "");
        boolean boolean9 = userRepository0.validateUser("hi!", "");
        boolean boolean12 = userRepository0.validateUser("", "hi!");
        boolean boolean15 = userRepository0.validateUser("hi!", "");
        boolean boolean18 = userRepository0.validateUser("", "");
        boolean boolean21 = userRepository0.validateUser("", "hi!");
        boolean boolean24 = userRepository0.validateUser("", "hi!");
        boolean boolean27 = userRepository0.validateUser("hi!", "hi!");
        org.junit.Assert.assertTrue("'" + boolean3 + "' != '" + false + "'", boolean3 == false);
        org.junit.Assert.assertTrue("'" + boolean6 + "' != '" + false + "'", boolean6 == false);
        org.junit.Assert.assertTrue("'" + boolean9 + "' != '" + false + "'", boolean9 == false);
        org.junit.Assert.assertTrue("'" + boolean12 + "' != '" + false + "'", boolean12 == false);
        org.junit.Assert.assertTrue("'" + boolean15 + "' != '" + false + "'", boolean15 == false);
        org.junit.Assert.assertTrue("'" + boolean18 + "' != '" + false + "'", boolean18 == false);
        org.junit.Assert.assertTrue("'" + boolean21 + "' != '" + false + "'", boolean21 == false);
        org.junit.Assert.assertTrue("'" + boolean24 + "' != '" + false + "'", boolean24 == false);
        org.junit.Assert.assertTrue("'" + boolean27 + "' != '" + false + "'", boolean27 == false);
    }
}

